### ISSUE TITLE
Confusion between sensu server and sensu client

### ISSUE THREAD

**Issue by basickarl** [sensu/sensu#1132]
I've been following the installation guide and it makes a right mess of the definition between the "server" and "client".

The following link displays an image: https://sensuapp.org/docs/0.21/overview
This page makes a clear definition as to what is what and the dependencies. However the installation guide makes a mess of this. I'm seeing docker images for example: https://github.com/usmanismail/docker-sensu-client which claims to be a sensu-client image which contains a redis database install because it it forks off of the server dockerfile (this could be due to A) laziness or B) lack of knowledge).

The following page: https://sensuapp.org/docs/latest/install-sensu-client states at the section "Install the Sensu client on remote hosts" the following:
```
Install Sensu Core
Please see: Install Sensu Core
```
The link takes the user to: https://sensuapp.org/docs/latest/install-sensu#install-sensu-core where it states:
```
Having successfully completed installation of Sensu’s dependencies (RabbitMQ, and Redis) 
```
However it was already stated in the overview that redis was not a dependency of a sensu client.

In other words the documentation is very confusing if you follow from a step-by-step process. I would recommend dividing the installation guide into two separate areas completely; "Sensu server" and "Sensu client".

**Comment by calebhailey** [comment]
@basickarl I couldn't agree with you more, which is why we have given the installation guide a _major_ rewrite with the 0.22 docs! See [here](https://sensuapp.org/docs/latest/installation-guide) for the new installation guide. 

As part of this rewrite, we've also started to explain in several different places that the Sensu Core _package_ (the installer), installs multiple processes, including `sensu-client`, `sensu-server`, and `sensu-api` (and some other utility packages like `sensu-install`); see [here](https://sensuapp.org/docs/latest/install-sensu-client#included-in-sensu-core). This has historically been a major point of confusion for new users, so we're hoping the new docs will help. 

The new installation guide also clarifies in multiple locations which services rely upon the [Sensu data store](https://sensuapp.org/docs/latest/data-store) (i.e. Redis) and [Sensu transport](https://sensuapp.org/docs/latest/transport) (RabbitMQ or Redis). We have also added some new pages called ["installation prerequisites"](https://sensuapp.org/docs/latest/installation-prerequisites) to help provide a big picture overview for how/where these dependencies are required. 

Aaaaaand if all of that seems a bit overwhelming, we agree (which isn't a good excuse, but it _is_ why we never went into so much detail before), we've also created a new "five minute install" guide, here: https://sensuapp.org/docs/latest/the-five-minute-install. 

I'm going to go ahead and close this issue under the assumption that these new docs will help fill in gaps in the old documentation, but please feel free to submit a new issue if you're still confused, here: https://github.com/sensu/sensu-docs/issues 

Finally, if you still need help learning Sensu, please consider [Sensu Training](https://sensuapp.org/training), or try hanging out in the #sensu Freenode channel - there are usually many Sensu users there who can help.

#monitoringlove
